tests/flow/tests_tensorflow.py

```diff
@@ -84,10 +84,6 @@ def test_run_mobilenet(env):
 
 @skip_if_no_TF
 def test_run_mobilenet_multiproc(env):
-    if VALGRIND:
-        env.debugPrint("skipping {} since VALGRIND=1".format(
-            sys._getframe().f_code.co_name), force=True)
-        return
```

Comment:
does it run in CI?

Reply:
Runs fine with valgrind locally, why not in CI? I'll know for sure when we merge to master....